src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -82,9 +86,28 @@ const defaultProps = {
     delegateAccountID: 0,
     actorIcon: {},
     isThreadParentMessage: false,
+    displayAsGroup: false,
 };
 
 function ReportActionItemFragment(props) {
+    /**
+     * Checks text element for presence of emoji as first character
+     * and insert Zero-Width character to avoid selection issue
+     * mentioned here https://github.com/Expensify/App/issues/29021
+     *
+     * @param {String} text
+     * @param {Boolean} displayAsGroup
+     * @returns {ReactNode | null} Text component with zero width character
+     */
+
+    function checkForEmojiForSelection(text, displayAsGroup) {
+        const firstLetterIsEmoji = EmojiUtils.firstLetterIsEmoji(text);
+        if (firstLetterIsEmoji && !displayAsGroup && !Browser.isMobile()) {
+            return <Text>&#x200b;</Text>;
+        }
+        return null;
+    }
```

Comment:
I would prefer this as an arrow function

Reply:
done, please check.